src/ipc-frontend-dbus.c

```diff
@@ -619,9 +622,17 @@ on_name_lost (GDBusConnection *connection,
               const gchar     *name,
               gpointer         user_data)
 {
-    g_info ("on_name_lost: %s", name);
+    g_debug ("%s: %s", __func__, name);
     IpcFrontend *ipc_frontend = IPC_FRONTEND (user_data);
+    IpcFrontendDbus *self = IPC_FRONTEND_DBUS (user_data);
+
+    if (self->connection_state == CONNECTION_STATE_DISCONNECTED) {
+        g_warning ("Failed to acquire DBus name %s. UID %d must be "
+                   "allowed to \"own\" this name. Check DBus config,",
```

Comment:
I don't think the comma at the end of the message log is correct. Otherwise users will wonder if there was some part of the sentence that wasn't printed.

Reply:
this was a typo, good catch